yal/docs/contact_fields.md

```diff
@@ -24,3 +24,41 @@ Fields
+| feedback_survey_sent                       | Sets whether the feedback has been sent                                                                 |
+| feedback_survey_sent_2                     | Sets whether the feedback has been sent                                                                 |
+| age                                        | Age of the user                                                                                         |
+| relationship_status                        | Relationship status of the user                                                                         |
+| gender                                     | Gender of the user                                                                                      |
+| location_description                       | Saves the `formatted_address` returned from the google location API                                     |
+| latitude                                   | saves the users latitudinal position                                                                    |
+| longitude                                  | saves the users longitudinal position                                                                   |
+| persona_name                               | saves the users preferred bot persona name                                                              |
+| persona_emoji                              | saves the users preferred bot persona emoji                                                             |
+| last_mainmenu_time                         | saves the last time the user was on the mainmenu, set in mainmenu                                       |
+| last_main_time                             | not being used - updated when `suggested_text` is updated, set in mainmenu                              |
+| suggested_text                             | not being used - a list of content that the user might be interested in to encourage engagement         |
+| privacy_reminder_sent                      | Set to True once the user has seen the main menu with a privacy message                                 |
+| push_related_page_id                       | the Id of the push message's related page, the id of the page that will be sent when the user requests to read more |
+| last_onboarding_time                       | saves the last time the user interacted with the onboarding flow                                        |
+| onboarding_reminder_type                   | saves which onboarding reminder must be saved, 5min, 2h, 8h                                             |
+| onboarding_reminder_sent                   | the onboarding reminder has been sent, resets when the user interacts with the onboarding reminder      |
+| callback_check_time                        | saves the time the user should be sent the callback check in and find out if they were called           |
```

Comment:
indicate that this callback is for the "please call me" or "chat to a counsellor" feature